surveys/2022-12/questions.md

```diff
@@ -0,0 +1,48 @@
+# Survey December 2022
+
+## Personal question
+
+- Which constituencies do you identify with?
+    - Direct end users: Users who run tools themselves
+    - Application operators: Users who interacts with existingm running applications
+    - Application developers: Frontend tools consumers, backend server authors, tools authors
+    - Library & package authors: Users who write librairies and packages to be included on other applications
+    - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
+    - Organization with investments in Node.js (eg: Enterprises, Gouvernement bodies, Startups, Non-profit)
```

Comment:
We should check if there were some that we added after the last survey, I think there was.